examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,535 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Streaming Responses\n",
+    "\n",
+    "Streaming allows you to process LLM responses in real-time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:16.508709Z",
+     "start_time": "2024-09-06T14:00:13.203020Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Tokyo, the capital city of Japan, is a vibrant metropolis known for its blend of traditional culture and cutting-edge modernity. With a population of over 13 million in the city proper and around 37 million in the Greater Tokyo Area, it is one of the most populous urban agglomerations in the world. Tokyo features a diverse range of neighborhoods, from the historic Asakusa with its ancient temples to the bustling shopping districts of Shibuya and Shinjuku.\n",
+      "\n",
+      "The city is renowned for its unique architecture, from contemporary skyscrapers to historic shrines and gardens. Tokyo offers a rich tapestry of cuisine, shopping, and entertainment, including world-class dining, bustling markets, and numerous parks. Its efficient public transportation system makes it easy to explore various attractions, such as the iconic Tokyo Tower, the peaceful Meiji Shrine, and the lively Akihabara district known for its otaku culture.\n",
+      "\n",
+      "Tokyo is also a major economic and cultural hub, hosting numerous international businesses, exhibitions, and events, making it a key player on the global stage."
+     ]
+    }
+   ],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\", stream=True)\n",
+    "@prompt_template(\"Provide a brief description of {city}.\")\n",
+    "def stream_city_info(city: str): ...\n",
+    "\n",
+    "\n",
+    "for chunk, _ in stream_city_info(\"Tokyo\"):\n",
+    "    print(chunk.content, end=\"\", flush=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information on streaming, including advanced techniques for processing streamed content, see our [Learn documentation on Streams](https://docs.mirascope.io/learn/streams)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Asynchronous Processing\n",
+    "\n",
+    "Mirascope supports asynchronous processing for efficient parallel execution of multiple LLM calls."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:17.154267Z",
+     "start_time": "2024-09-06T14:00:16.512749Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The capital of France is The capital of France is Paris.\n",
```

Comment:
the output here is a little weird because the LLM is including the text you're also including.

I think we could improve this by either only printing the content or instead using `response_model` here.

I'm thinking we should put Response Models before this section and then use `response_model` for the async example (and focus on extraction rather than generation). Thoughts?